src/engine/duel/ai/core.asm

```diff
@@ -1308,14 +1323,18 @@ Func_15886:
 
 INCLUDE "engine/duel/ai/retreat.asm"
 
-; Copy cards from wDuelTempList in hl to wHandTempList in de
-CopyHandCardList:
+; copies an $ff-terminated list from hl to de.
+; preserves bc
+; input:
+;	hl = address from which to start copying the data
+;	de = where to copy the data
+CopyListWithFFTerminatorFromHLToDE_Bank5:
```

Comment:
I feel like this is kind of a mouthful compared to the previous name. IMO we could exclude "WithFFTerminator" from the name.

Reply:
That one's tricky. I only added the "WithFFTerminator" because there's already a `CopyListFromHLToDE` in src/engine/menus/deck_configuration.asm (Bank $02), and that function requires a null-terminated list. If you have any suggestions now that you know the problem, I'm all ears.